### DISCUSSION TITLE
Can't find any information on how to add a custom skin (osk) to osu running on linux (osu.appimage) ?

### DISCUSSION THREAD

**Discussion by AbrieJCronje** [ppy/osu#13495] [Q&A] [answered]
Would be very helpful if there is some resource out there to have a link to it on the readme. 

Think changing skins is one of the first things regular players want to do?

**Comment by bdach** [comment, marked as answer]
To import a skin, drag and drop it into the game window, or supply the path to the skin archive as a command-line argument via a shell.

**Reply by Susko3** [reply]
Alternatively, one can also use the `Import files` button in settings.

**Reply by AbrieJCronje** [reply]
@Susko3 Thanks this is definitely the easiest way, dragging it into the game window did nothing for me.

**Reply by bdach** [reply]
@AbrieJCronje I use linux and I can assure you that drag & drop works here. If it doesn't for you, then please open an issue about it this time, because that would actually be a bug.

**Reply by peppy** [reply]
not sure an issue should be opened, 99% user's issue when it comes to linux.

**Reply by gagahpangeran** [reply]
@AbrieJCronje what's your desktop environment or window manager?

I use plain i3 and drag & drop works for me.
